app/javascript/packs/building_map.js

```diff
@@ -0,0 +1,73 @@
+import { displayRoute, setupMap } from './leafletMap.js';
+
+let currentLocation;
+const YOUR_LOCATION_MAGIC_STRING = "Your location" // TODO: Change this!!!
```

Comment:
This is because we cannot execute Ruby code inside the JS file, right? It might be difficult to fix (and use the Ruby constant again), because afaik there is no such thing as a `.js.erb`.

Reply:
We can't really pass things from ruby over to js. We might find a workaround, where we provide this thing from the server via an ajax request (like we did with the other constants) but maybe it won't be necessary due to the new UI which uses js as well?

Reply:
AJAX just for getting a string constant seems a bit ugly (because we need new/separate endpoints for each constant), but I'm fine with leaving this as it is until we know how the new UI and translation is implemented. We should just keep this in mind.

Reply:
Instead of using an Ajax call to get a string, you could also consider writing the value as a `data` attribute into an HTML element in the `.erb` template, which is read by the library JS, i.e. `<div class="map" data-location="Your location"></div>`. Then the value is bound to the element that requires it. This pattern is sometimes called 'Unobtrusive JS': https://guides.rubyonrails.org/working_with_javascript_in_rails.html#unobtrusive-javascript